### PR TITLE
BREAKING CHANGE: parse `<=` as `log:isImpliedBy`

### PR DESCRIPTION
**Pull request by jeswr**

### CODE REVIEW

File: test/N3Parser-test.js [hunk 1]
```diff
@@ -1433,6 +1433,353 @@ describe('Parser', () => {
   describe('A Parser instance for the N3 format', () => {
     function parser() { return new Parser({ baseIRI: BASE_IRI, format: 'N3' }); }
 
+    it('should parse a single triple',
+      shouldParse(parser, '<a> <b> <c>.', ['a', 'b', 'c']));
+
+    it('should not parse a default graph',
+      shouldNotParse(parser, '{}', 'Expected entity but got eof on line 1.'));
```
Comment (TallTed) [inline test/N3Parser-test.js:1440]:
```suggestion
      shouldNotParse(parser, '{}', 'Expected entity but got EOF on line 1.'));
```

File: test/N3Parser-test.js [hunk 2]
```diff
@@ -1433,6 +1433,353 @@ describe('Parser', () => {
...
+            ['"bonjour"@fr', 'sameAs', '"hello"@en', '_:b0']
+        ));
+
+    it('should not parse RDF* in the subject position',
+      shouldNotParse(parser, '<<<a> <b> <c>>> <a> <b> .',
+        'Unexpected RDF* syntax on line 1.'));
+
+    it('should not parse RDF* in the object position',
+      shouldNotParse(parser, '<a> <b> <<<a> <b> <c>>>.',
+        'Unexpected RDF* syntax on line 1.'));
```
Comment (TallTed) [inline test/N3Parser-test.js:1777]: Though `RDF*` and `RDF-star` are pronounced the same, they should not be used interchangeably. `RDF*` identifies an old, draft idea. `RDF-star` identifies the current work-in-progress which is expected to be standardized in the next couple of years (Community Group [report](https://www.w3.org/2021/12/rdf-star.html), Working Group [repositories](https://www.w3.org/groups/wg/rdf-star/tools)). There are MAJOR differences between these, and confusion among them should be avoided whenever possible. I *believe* the intent here is `RDF-star`, but I could be wrong, so please review the linked documents, and make appropriate adjustments.
```suggestion
    it('should not parse RDF-star in the subject position',
      shouldNotParse(parser, '<<<a> <b> <c>>> <a> <b> .',
        'Unexpected RDF-star syntax on line 1.'));

    it('should not parse RDF-star in the object position',
      shouldNotParse(parser, '<a> <b> <<<a> <b> <c>>>.',
        'Unexpected RDF-star syntax on line 1.'));
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: test/N3Parser-test.js (hunk 1 above)
Comment (jeswr) [reply test/N3Parser-test.js:1440]: This is testing and existing error message in the parser.

File: test/N3Parser-test.js (hunk 2 above)
Comment (jeswr) [reply test/N3Parser-test.js:1777]: This PR targets the main branch which currently implements something closer to `RDF*`. These references will be updated when merged with the changes in https://github.com/rdfjs/N3.js/pull/311 which implements `RDF-star` and uses such terminology in its error messaging.

**Comment by jeswr** [comment]
Closing in favour of #341

**Pull request closed without merging**